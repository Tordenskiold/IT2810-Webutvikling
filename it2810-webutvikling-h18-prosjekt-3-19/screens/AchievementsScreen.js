import React from 'react';
import { ScrollView, StyleSheet } from 'react-native';
import Achievements from '../components/Achievements';

export default class AchievementsScreen extends React.Component {
  static navigationOptions = {
    title: 'Achievements',
  };

  render() {
    return (
      <ScrollView style={styles.container}>
        <Achievements></Achievements>
      </ScrollView>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    // paddingTop: 15,
    backgroundColor: '#fff',
  },
});
