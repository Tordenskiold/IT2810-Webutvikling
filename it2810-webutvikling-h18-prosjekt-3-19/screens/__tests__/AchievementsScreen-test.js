import React from "react";
import AchievementsScreen from "../AchievementsScreen";
import renderer from "react-test-renderer";

test("AchievementsScreen renders correctly", () => {
  const tree = renderer.create(<AchievementsScreen />);
  expect(tree.toJSON()).toMatchSnapshot();
});
