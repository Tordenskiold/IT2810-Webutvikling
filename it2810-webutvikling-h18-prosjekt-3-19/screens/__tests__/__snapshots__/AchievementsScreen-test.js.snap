// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AchievementsScreen renders correctly 1`] = `null`;
